pandas/io/parsers.py

```diff
@@ -3193,8 +3194,7 @@ def _rows_to_cols(self, content):
                         i < len(self.index_col)
                         # pandas\io\parsers.py:3159: error: Unsupported right
                         # operand type for in ("Optional[Any]")  [operator]
-                        or i - len(self.index_col)  # type: ignore[operator]
-                        in self._col_indices
+                        or i - len(self.index_col) in self._col_indices
```

Comment:
Since you do not ignore mypy checking anymore, then the comments above are irrelevant.
Maybe xref #37715?

Reply:
Thx, that is a good point.